end-to-end-tests/sdk-set-user-id.test.js

```diff
@@ -9,7 +9,7 @@ beforeAll(() => {
     .then(killFunc => {
       killServer = killFunc;
     })
-    .then(() => createChromeWebDriver())
+    .then(() => createChromeWebDriver(NOTIFICATIONS_GRANTED))
```

Comment:
If this is `NOTIFICATIONS_GRANTED` everywhere - should that just be the default?

Reply:
I'm not sure. It felt wrong to have `NOTIFICATIONS_DEFAULT` (which matches the browser state) not be the default. I don't think it's bad this way. Makes it more explicit that the browser will have notifications granted, which wouldn't normally be the default.